docs/proposals/controller/README.md

```diff
@@ -0,0 +1,38 @@
+# Cluster Controller
+
+The Cluster Controller is an intermediary between the client and scheduler, its main task is to compare two states: the one requested by the client and the existing one reported by the scheduler and saved in the database. Based on the comparison results, the controller sends requests to the scheduler API in order to transform the current state into the requested one.
+
+### Services provided:
+
+HTTP API (OpenAPI 3.0 compliant) to connect with CLI
+gRPC client and API for the scheduler
+DATABASE access and control
```

Comment:
A database is an internal implementation choice. Also, this is not an exposed service. What you want is a consistent, non-volatile way of tracking the cluster state. It can be done through a database or something else, it's an implementation detail.

Reply:
Noted. We are removing the database from the services list then.